src/app/weather-data/__snapshots__/weather-data.component.spec.ts.snap

```diff
@@ -3,10 +3,13 @@
 exports[`WeatherDataComponent Should match snapshot 1`] = `
 <app-weather-data
   __ngContext__="0"
-  data={[Function Object]}
+  dataLocation={[Function Object]}
+  dataWTime={[Function Object]}
```

Comment:
What does the W stand for?

Reply:
It was meant to be standing for "with" like how you would use "w/"

Reply:
Ok. 😄